src/librustc/mir/visit.rs

```diff
@@ -383,7 +383,7 @@ macro_rules! make_mir_visitor {
                         for output in & $($mutability)* outputs[..] {
                             self.visit_place(output, PlaceContext::AsmOutput, location);
                         }
-                        for input in & $($mutability)* inputs[..] {
+                        for (_, input) in & $($mutability)* inputs[..] {
                             self.visit_operand(input, location);
```

Comment:
This needs a `self.visit_span`.

Reply:
Fixed. I've force pushed to avoid another CI check when I squash the commits...